data/transactions/verify/txn_test.go

```diff
@@ -445,3 +444,78 @@ func BenchmarkTxn(b *testing.B) {
 	}
 	b.StopTimer()
 }
+
+// TestTxnGroupCacheUpdate uses TxnGroup to verify txns and add them to the
+// cache. Then makes sure that only the valid txns are verified and added to
+// the cache.
+func TestTxnGroupCacheUpdate(t *testing.T) {
+	partitiontest.PartitionTest(t)
```

Comment:
Since this presumably doing slow crypto ops, can we do `t.Parallel()`?

Reply:
Depending on what runs with it, things can get flakey